ditaa-ditaa.rb

```diff
@@ -2,6 +2,13 @@
 require "tempfile"
 require "digest/sha1"
 
+begin
+	require "fastimage"
+rescue LoadError
+	STDERR.puts "You are missing a library required for retina image parsing in ditaa-ditaa. Please run:"
```

Comment:
Line is too long. [102/80]<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.